### ISSUE TITLE
You cannot hangup an incoming call

### ISSUE THREAD

**Issue by sadaf12345** [ASWATFZLLC/verto-tuto#3]
1) Open /verto-tuto/basic-integration/index.html

2) Connect to Position.

3) Connect the Agent.

4) Call Click2call API

5) Call DID

6) I recieve the call to /verto-tuto/basic-integration/index.html

7) Click Hangup

The call keeps ringing.

Console log:
jquery.verto.js:2146 Dialog cb1627c2-5dd2-4c7a-bfec-0e491c6d6270: state change from ringing to hangup
jquery.verto.js:2146 Dialog cb1627c2-5dd2-4c7a-bfec-0e491c6d6270: state change from hangup to destroy
jquery.verto.js:2146 Dialog 30a67d77-5e20-4cfd-af38-4cde2a08c446: state change from new to ringing

8) Click Hangup again

The call keeps ringing.

Console log:
jquery.verto.js:2146 Dialog 30a67d77-5e20-4cfd-af38-4cde2a08c446: state change from ringing to hangup
jquery.verto.js:2146 Dialog 30a67d77-5e20-4cfd-af38-4cde2a08c446: state change from hangup to destroy
jquery.verto.js:2146 Dialog 30a67d77-5e20-4cfd-af38-4cde2a08c446: state change from new to ringing

Kindly advise as how to resolve these issues.

**Comment by CaptainYouz** [comment]
This is probably related to the Queue strategy. If the Queue strategy is ring-all, or if the Agent is the only one attached to the queue, it will keep sending the call again and again until the caller hangs up the call.

**Comment by sadaf12345** [comment]
@CaptainYouz Is there possibility where Agent can cancel the call. Like any function which we can refer to the Customer from verto library?

**Comment by CaptainYouz** [comment]
No, since the call comes from Freeswitch, it is Freeswitch that automatically send calls to Agent concerned.
